examples/configs/autoalign/autoalign_config/config.yml

```diff
@@ -0,0 +1,73 @@
+models:
+  - type: main
+    engine: openai
+    model: gpt-3.5-turbo
+    parameters:
+      temperature: 0.0
+rails:
+    config:
+        autoalign:
+            parameters:
+                endpoint: "https://nvidia.autoalign.ai/guardrail"
```

Comment:
These URLs should be changed to generic placeholders `"<AUTOALIGN_ENDPOINT>"`. Or `"https://<ACCOUNT>.autoalign.ai/guardrail"`.

Reply:
resolved in 2b446091693e44cb629d0cd79223a9519df1cdd1